server/sonar-web/src/main/js/apps/issues/components/AppContainer.js

```diff
@@ -47,16 +47,22 @@ const fetchIssueOrganizations = issues => dispatch => {
   );
 };
 
-const fetchIssues = (query /*: RawQuery */) => dispatch =>
-  searchIssues({ ...query, additionalFields: '_all' })
+const fetchIssues = (query /*: RawQuery */) => (dispatch, getState) => {
+  const organizationsEnabled = areThereCustomOrganizations(getState());
+  return searchIssues({ ...query, additionalFields: '_all' })
     .then(response => {
       const parsedIssues = response.issues.map(issue =>
         parseIssueFromResponse(issue, response.components, response.users, response.rules)
       );
       return { ...response, issues: parsedIssues };
     })
-    .then(response => dispatch(fetchIssueOrganizations(response.issues)).then(() => response))
+    .then(response => {
+      return organizationsEnabled
```

Comment:
So the organization is not fetched anymore when organization are not enabled, but we still fetch the default organization for every facet that the use open. Is it intended to stay like this ?